<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="DESKTOP-8L83O0F" name="com.qa.hubspot.tests.HomePageTest" tests="4" failures="0" timestamp="01 Jul 2020 17:38:07 GMT" time="0.015" errors="1">
  <testcase name="setUp" time="0.015" classname="com.qa.hubspot.tests.HomePageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.BasePage.initialize_driver(BasePage.java:22)
at com.qa.hubspot.tests.HomePageTest.setUp(HomePageTest.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="verifyHomePageTitleTest" time="0.000" classname="com.qa.hubspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyHomePageTitleTest -->
  <testcase name="verifyHomePageHeaderTest" time="0.000" classname="com.qa.hubspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyHomePageHeaderTest -->
  <testcase name="verifyLoggedInUserAccountName" time="0.000" classname="com.qa.hubspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyLoggedInUserAccountName -->
</testsuite> <!-- com.qa.hubspot.tests.HomePageTest -->
